packages/api-providers/src/abstracts/anchor/anchor-deposit.ts

```diff
@@ -6,10 +6,14 @@ import { WebbApiProvider } from '../webb-provider.interface.js';
 
 export type AnchorSize = MixerSize;
 
-export abstract class AnchorDeposit<
-  T extends WebbApiProvider<any>,
-  K extends DepositPayload = DepositPayload<any>
-> extends MixerDeposit<T, K> {
+// Todo: should we extract the interface of MixerDeposit on another class and rename `generateBridgeNote` to generate note?
```

Comment:
```suggestion
// Todo: should we extract the interface of MixerDeposit on another class and rename `generateBridgeNote` to generate note
```
Agreed this refactor should take place - especially with the removal of tornado.